lms/views/lti/basic_launch.py

```diff
@@ -248,8 +248,11 @@ def _show_document(self, assignment):
         # If there are any Hypothesis client feature flags that need to be
         # enabled based on the current application instance settings, those
         # should be enabled here via `self.context.js_config.enable_client_feature`.
-        #
-        # There are currently no such features.
+        ai_settings = self.request.lti_user.application_instance.settings
+        if ai_settings.get_setting(
+            ai_settings.fields[ai_settings.Settings.HYPOTHESIS_MENTIONS]
+        ):
+            self.context.js_config.enable_client_feature("at_mentions")
```

Comment:
Once the setting to enable email address collection is added, we should add that condition here as well.

So, the `at_mentions` client feature flag should be enabled only if both hypothesis mentions and email collections are enabled in the instance.

Reply:
@acelaya Added a commit to this effect c3b9b7ec65a7e1c870644623e965a30eb31dc802

Worth taking a second look in case I made a mistake